resources/task.rb

```diff
@@ -43,6 +43,7 @@
 attribute :start_day, kind_of: String, default: nil
 attribute :start_time, kind_of: String, default: nil
 attribute :day, kind_of: [String, Integer], default: nil
+attribute :months, kind_of: String, default: nil
```

Comment:
Yeah I took a look at the windows `schtasks` docs and I think day is meant to take just one day: 
```
    /D    days         Specifies the day of the week to run the task. Valid
                       values: MON, TUE, WED, THU, FRI, SAT, SUN and for
                       MONTHLY schedules 1 - 31 (days of the month).
                       Wildcard "*" specifies all days.

    /M    months       Specifies month(s) of the year. Defaults to the first
                       day of the month. Valid values: JAN, FEB, MAR, APR,
                       MAY, JUN, JUL, AUG, SEP, OCT, NOV, DEC. Wildcard "*"
                       specifies all months.
```
The `/M` flag mentions "month(s)" while `/D` just says "day".